tools/cuttlefish-host-image-installer/metapackage-google/debian/metapackage-google.cron.d

```diff
@@ -3,3 +3,4 @@
 #
 # Workarounds for Network-Manager
 @reboot root sleep 100 && /usr/sbin/dhclient
+@reboot root sleep 150 && su vsoc-01 -c '/bin/bash /usr/share/metapackage-google/cf_docker_co_run.sh'
```

Comment:
OOC: Does `cf_docker_co_run.sh` run as daemon?

Reply:
yes, this script "cf_docker_co_run.sh" keeps listening to the incoming ports and never stop.

Reply:
vsoc-01@arm-018:~/cloud-android-orchestration-0.1.0-alpha$ ps aux | grep run.sh
root        1925  0.0  0.0   2316   812 ?        Ss   16:51   0:00 /bin/sh -c sleep 150 && su vsoc-01 -c '/bin/bash /usr/share/metapackage-google/cf_docker_co_run.sh'
root        3245  0.0  0.0   9868  3456 ?        S    16:53   0:00 su vsoc-01 -c /bin/bash /usr/share/metapackage-google/cf_docker_co_run.sh
vsoc-01     3263  0.0  0.0   6796  3156 ?        Ss   16:53   0:00 /bin/bash /usr/share/metapackage-google/cf_docker_co_run.sh
vsoc-01    34152  0.0  0.0   6796  3156 ?        S    16:58   0:00 /bin/bash scripts/docker/run.sh
vsoc-01    44051  0.0  0.0   6084  1980 pts/0    S+   17:02   0:00 grep run.sh
vsoc-01@arm-018:~/cloud-android-orchestration-0.1.0-alpha$

Reply:
Ah, I see. I wasn't familiar for such implementation, so I wanted to know that. Thanks!